CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cmake_policy(SET CMP0057 NEW)
 
 # bpftrace version number components.
 set(bpftrace_VERSION_MAJOR 0)
-set(bpftrace_VERSION_MINOR 23)
+set(bpftrace_VERSION_MINOR 25)
```

Comment:
Why 25? We haven't released 0.25.0, yet.

Reply:
This was more of a conversation starter. What is the "version" of the master branch? Thinking out loud...as soon as you cut a release branch (in this case 0.24) then master version should also be incremented (something like  0.25.0-prerelease or 0.25.0-dev to indicate this code is for the NEXT version but it's not production ready).

Reply:
>  let me formalize this in the release guide and we can discuss details in the PR.

Sure, I'll close this PR for now but I think I prefer the later option 😉